src/JitInterface/src/JitConfigProvider.cs

```diff
@@ -7,7 +7,7 @@
 using System.Diagnostics;
 using System.Runtime.InteropServices;
 using System.Threading;
-
+using ILCompiler;
```

Comment:
Is this `using` needed?

Reply:
No. It came from the runtime repo though. Fixing them adds friction to the integration. I wish we had a two way automatic mirror like in the old times...